frontend/public/components/limit-range.tsx

```diff
@@ -0,0 +1,174 @@
+  <ListHeader>
+    <ColHead {...props} className="col-xs-2" sortField="resourceType">Resource Type</ColHead>
+    <ColHead {...props} className="col-xs-2" sortField="min">Min</ColHead>
+    <ColHead {...props} className="col-xs-2" sortField="max">Max</ColHead>
+    <ColHead {...props} className="col-xs-2" sortField="defaultRequest">Default Request</ColHead>
+    <ColHead {...props} className="col-xs-2" sortField="default">Default Limit</ColHead>
+    <ColHead {...props} className="col-xs-2" sortField="maxLimitRequestRatio">Max Limit/Request Ratio</ColHead>
+  </ListHeader>;
+
+const LimitRangeDetailsRow: React.SFC<LimitRangeRowProps> = ({obj}) =>
+  <div className="row co-resource-list__item">
+    <div className="col-xs-2">
+      {obj.resourceType}
+    </div>
+    <div className="col-xs-2">
+      {obj.min || '-'}
+    </div>
+    <div className="col-xs-2">
+      {obj.max || '-'}
+    </div>
+    <div className="col-xs-2">
+      {obj.defaultRequest || '-'}
+    </div>
+    <div className="col-xs-2">
+      {obj.default || '-'}
+    </div>
+    <div className="col-xs-2">
+      {obj.maxLimitRequestRatio || '-'}
+    </div>
+  </div>;
+
+export const LimitRangeDetailsList = (resource) => {
+  const data = [];
+
+  for (let limitRange of resource.resource.spec.limits) {
+    let type = limitRange.type; //We have nested types, top level type is something like "Container"
+    let limitRangeProperties = ['max', 'min', 'default', 'defaultRequest', 'maxLimitRequestRatio'];
+    let cpuLimits = {};
+    let memoryLimits = {};
+    for (let i = 0; i<limitRangeProperties.length; i++) {
+      let limitRangeProperty = limitRangeProperties[i];
+      if (limitRange.hasOwnProperty(limitRangeProperty)){
+        if (limitRange[limitRangeProperty].hasOwnProperty('cpu')){
+          let value = limitRange[limitRangeProperty].cpu;
+          if (limitRangeProperty !== 'maxLimitRequestRatio'){
+            // make sure cpu values have the correct units
+            // m = millicores, '' = cores
+            if (value.slice(-1) === 'm') {
+              let number = value.slice(0, -1);
+              value = `${number} millicores`;
+            } else {
+              value += ' cores';
+            }
+          }
+          cpuLimits[limitRangeProperty] = value;
+        }
+        if (limitRange[limitRangeProperty].hasOwnProperty('memory')){
+          let value = limitRange[limitRangeProperty].memory;
+          if (limitRangeProperty !== 'maxLimitRequestRatio') {
+            // make sure memory values have the correct units
+            // Mi = MiB, Gi = GiB, M = MB, G = GB
+            value += 'B';
+          }
+          memoryLimits[limitRangeProperty] = value;
+        }
+      }
+    }
+    if (!_.isEmpty(cpuLimits)){
+      cpuLimits.resourceType = `${type} CPU`;
+      data.push(cpuLimits);
+    }
+    if (!_.isEmpty(memoryLimits)){
+      memoryLimits.resourceType = `${type} Memory`;
+      data.push(memoryLimits);
+    }
+  }
+  return <List
```

Comment:
For tables that aren't kubernetes resources, we typically don't use the `List` component. We just create the table. For instance:

https://github.com/openshift/console/blob/master/frontend/public/components/pod.jsx#L139

I'd have to look at this more to see if we should consider List for other kinds of tables.